### DISCUSSION TITLE
采用 Release 创建发行版

### DISCUSSION THREAD

**Discussion by imfing** [whutug/whu-thesis#42] [Ideas]
用 GitHub 的 Release 发布和更新版本，便于迭代。

参考：
* https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository
* https://github.com/microsoft/vscode/releases

**Comment by tanukihee** [comment]
最开始是想到一个稳定版本（v1.0）再开始发布release并详细记录更新日志的，现在看着似乎也没什么东西可以加，可以试试了

**Reply by tanukihee** [reply]
现在还有一个开发方向，用 xtemplate 把整个模板面向对象化，然后把所有格式上可变的都拿出来，也许放进 config 文件里

完成这两个估计就可以发布正式版了

**Reply by imfing** [reply]
我建议可以先直接发布0.x版本的 然后在这个基础上迭代。
至于xtemplate的开发方向，可以先弄到 dev 分支里面

**Reply by tanukihee** [reply]
把版本号提到 v0.6，发行了第一个版本

好懒，不想写 changelog……🤣🤣
